records/generator.go

```diff
@@ -543,6 +553,66 @@ func (rg *RecordGenerator) insertRR(name, host string, kind rrsKind) (added bool
 	return
 }
 
+func rrsKindForIP(ip net.IP) rrsKind {
+	if ip.To4() != nil {
+		return A
+	} else if ip.To16() != nil {
+		return AAAA
+	}
+	return A
```

Comment:
should this panic instead? it's unlikely that we'd ever end up here but I hate silent failures

Reply:
I agree, my only concern is that there are a bunch of tests with master "ip addresses" like `6`, or `bob`, etc. I assume the tests are just flawed and I can make them into valid ipv4 addresses to pass the tests, but I'm not sure if there's a valid use case for masters with non-ip hostnames in master pids / addresses. Should we maybe just not create records and error log for those cases? I'm also fine with panic.